code/src/components/Header.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import styled from 'styled-components'
+
+const Header = () => {
+  return (
+    <Title>TASKS TO DO</Title>
+    );
+}
+
```

Comment:
Great to simplify and split up to several components! A suggestion would be to make this even more generic by for example put different styled text-elements in a folder called "lib" and then you could for example import this header from there in to your TopSection-component.

Reply:
Nice suggestion.